app/models/katello/repository.rb

```diff
@@ -130,6 +134,7 @@ class Repository < Katello::Model
     scope :docker_type, -> { with_type(DOCKER_TYPE) }
     scope :ostree_type, -> { with_type(OSTREE_TYPE) }
     scope :ansible_collection_type, -> { with_type(ANSIBLE_COLLECTION_TYPE) }
+    scope :generic_content_type, -> { with_type(GENERIC_CONTENT_TYPE) }
```

Comment:
This should be `:generic_type`.

Reply:
I also just realized that `with_type()` is returning `[]` becuase the Python repo's content_type is 'python' instead of 'generic'.  I think we could edit the scope with some kind of query for all generic repositories without using `with_type`.

Reply:
> I also just realized that `with_type()` is returning `[]` becuase the Python repo's content_type is 'python' instead of 'generic'. I think we could edit the scope with some kind of query for all generic repositories without using `with_type`.

[I tried something](https://github.com/Katello/katello/blob/3832fe23d095bf9964f58368bcb3321cb535a0ee/app/models/katello/repository.rb#L137), not sure about it though. Seems okay if I compare it to the result of with_type on other types.